image_generation/requirements.txt

```diff
@@ -0,0 +1,6 @@
+--extra-index-url https://download.pytorch.org/whl/cpu
+torch==2.2.2+cpu; sys_platform != 'darwin'
+torch==2.2.2; sys_platform == 'darwin'  # Torch wheel for 2.2.2+cpu does not exist for macOS, using 2.2.2 instead
+diffusers==0.27.2
+optimum-intel[openvino]==1.18.0
```

Comment:
```suggestion
```
samples already have `optimum[openvino]==1.20.0`. That's strange that pip didn't detect a conflict

Reply:
From samples optimum-intel is installed with version >=1.16.0, so there is no conflict.
But I removed duplicated dependencies from image generation